src/main/java/com/ft/methodearticlemapper/resources/PostContentToTransformResource.java

```diff
@@ -61,28 +72,28 @@ public final Content doTransform(@PathParam("uuidString") String uuid, @QueryPar
 					.reason(ErrorMessage.METHODE_FILE_NOT_FOUND)
 					.exception(e);
 		}catch(NotWebChannelException e){
-			throw ClientError.status(404)
+			throw ClientError.status(422)
 					.reason(ErrorMessage.NOT_WEB_CHANNEL)
 					.exception(e);
 		}catch(MethodeMissingFieldException e){
-			throw ClientError.status(404)
+			throw ClientError.status(422)
 					.error(String.format(ErrorMessage.METHODE_FIELD_MISSING.toString(), e.getFieldName()))
 					.exception(e);
 		}catch (MethodeMissingBodyException | UntransformableMethodeContentException e) {
-			throw ClientError.status(418)
+			throw ClientError.status(422)
 					.error(e.getMessage())
 					.exception(e);
 		}catch(MethodeContentNotEligibleForPublishException e){
-			throw ClientError.status(404)
-					.context(uuid.toString())
+			throw ClientError.status(422)
+					.context(uuid)
 					.error(e.getMessage())
 					.exception(e);
 		}
 	}
 
 	private void validateUuid(String uuid, EomFile eomFile) {
 		if (uuid == null) {
-			throw ClientError.status(400).context(uuid).reason(ErrorMessage.UUID_REQUIRED).exception();
+			throw ClientError.status(400).context(null).reason(ErrorMessage.UUID_REQUIRED).exception();
```

Comment:
Shouldn't this return 422? I think PAM will not be able to issue a publish failure for status 400.